src/core/js/device.js

```diff
@@ -187,8 +187,9 @@ define(function(require) {
         }
     }
 
-    // Convert 'msie' to 'ie' for backwards compatibility
-    var browserString = (Adapt.device.browser.toLowerCase() === 'msie') ? 'ie' : Adapt.device.browser.toLowerCase();
+    var browser = Adapt.device.browser.toLowerCase();
+    // Convert 'msie' and 'internet explorer' to 'ie'.
+    var browserString = browser.replace(/^msie$|^internet explorer$/, 'ie');
```

Comment:
Could we perhaps lose the string boundary tokens? I personally find it more readable without and less need to refer back to a regex reference.
```js
var browserString = browser.replace(/msie|internet explorer/, 'ie');
```